packages/api/src/providers/google-calendar/utils.ts

```diff
@@ -85,20 +85,42 @@ export function parseGoogleCalendarEvent({
     accountId,
     calendarId: calendar.id,
     readOnly: calendar.readOnly,
+    conferenceData: parseGoogleCalendarConferenceData(event.conferenceData),
   };
 }
 
 export function toGoogleCalendarEvent(
   event: CreateEventInput | UpdateEventInput,
 ): GoogleCalendarEventCreateParams {
-  return {
+  console.log({
+    conferenceData: JSON.stringify(event.conferenceData, null, 2),
+  });
+
+  const result: GoogleCalendarEventCreateParams = {
     ...("id" in event ? { id: event.id } : {}),
-    summary: event.title,
-    description: event.description,
-    location: event.location,
     start: toGoogleCalendarDate(event.start),
     end: toGoogleCalendarDate(event.end),
+    conferenceDataVersion: 1, // This ensures the conference data is created, DO NOT REMOVE
   };
+
+  // Only include fields that are actually provided to avoid overwriting with undefined
+  if (event.title !== undefined) {
+    result.summary = event.title;
+  }
+
+  if (event.description !== undefined) {
+    result.description = event.description;
+  }
+
+  if (event.location !== undefined) {
+    result.location = event.location;
+  }
+
+  if (event.conferenceData !== undefined) {
+    result.conferenceData = event.conferenceData;
+  }
```

Comment:
we want to override things with undefined.

otherwise, how can you clear fields?